### ISSUE TITLE
[Test Framework] Support TLS is Native Echo component

### ISSUE THREAD

**Issue by nmittler** [istio/istio#13778]

**Comment by howardjohn** [comment]
@nmittler is this done now that we have docker?

**Comment by nmittler** [comment]
@howardjohn echo *should* support it now, but I think there may still be some pieces missing (native citidel?) before we can make sure it works.

**Comment by howardjohn** [comment]
Effectively duplicate of https://github.com/istio/istio/issues/9177
